### DISCUSSION TITLE
What's rsdw-server?

### DISCUSSION THREAD

**Discussion by SunHuawei** [wakujs/waku#242] [Q&A]
Not any results from Google and npmjs.com. It seems to replace `react-server-dom-webpack/server.edge` but for PROD. How can I find the source code?

**Comment by dai-shi** [comment]
As you said, it's a production bundle for `react-server-dom-webpack/server.edge`.

The code at npmjs is here: https://www.npmjs.com/package/react-server-dom-webpack/v/18.3.0-canary-5bcade5fc-20231208?activeTab=code -> `/react-server-dom-webpack/cjs/react-server-dom-webpack-server.edge.production.js`

Or here: https://unpkg.com/browse/react-server-dom-webpack@18.3.0-canary-5bcade5fc-20231208/cjs/react-server-dom-webpack-server.edge.production.js

Or directly to github: https://github.com/facebook/react/blob/5bcade5fcf5610e82e7cda05cc6de574bdace0c7/packages/react-server-dom-webpack/src/ReactFlightDOMServerEdge.js

You can't google it. It's such a new thing.
